dvc/lock.py

```diff
@@ -15,6 +15,12 @@
 
 DEFAULT_TIMEOUT = 5
 
+ERROR_MESSAGE = (
+    "cannot perform the command since other dvc process is running for this "
```

Comment:
dvc -> DVC

@jorgeorpinel please take a look once again

Reply:
I think we could use back ticks in this case since lower case `dvc` is the name of the process. I'll leave a separate comment on this around the whole string for easier readability.